cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -164,7 +192,6 @@ const Modules = () => {
                 Path:
                 <Link
                   aria-level={3}
-                  target="_blank"
```

Comment:
Can you revert this part?

Reply:
@nikhiltanna can you also revert this one

Reply:
Done